source/_integrations/pglab.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: PG LAB Electronics
+description: Instructions on how to integrate PG LAB Electronics with Home Assistant.
+ha_category:
+  - Switch
+ha_release: "2025.2"
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - "@pglab-electronics"
+ha_domain: pglab
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+This integration allows you to control [PG LAB Electronics](https://www.pglab.dev/) devices in Home Assistant.
```

Comment:
```suggestion
The **PG LAB Electronics** {% term integrations %} allows you to control [PG LAB Electronics](https://www.pglab.dev/) devices in Home Assistant.
```
For all the terms defined in the [glossary](https://www.home-assistant.io/docs/glossary/), you can add a glossary tooltip.